pyfakefs/fake_filesystem.py

```diff
@@ -3440,6 +3445,22 @@ def islink(self, path: AnyPath) -> bool:
         """
         return self._is_of_type(path, S_IFLNK, follow_symlinks=False)
 
+    def isjunction(self, path: AnyPath) -> bool:
+        """Determine if path identifies a junction.
+
+        Args:
+            path: Path to filesystem object.
+
+        Returns:
+            `False` on posix systems.
+            `True` if path is a junction on Windows.
+
+        Raises:
+            TypeError: if path is None.
+        """
+        # TODO: implement junction on Windows
+        return False
```

Comment:
I think for the time being it is ok to just return `False` here - I would adapt the description accordingly, though. Later, we may add some convenience function to make a junction in `pyfakefs` - as far as I can see, this is not possible in Python yet. We also need to adapt `os.stat`, `os.lstat` and possibly other functions that will support junctions.